M2/Macaulay2/packages/DGAlgebras.m2

```diff
@@ -1,26 +1,64 @@
--- -*- coding: utf-8 -*-
+-*- coding: utf-8 -*-
 newPackage("DGAlgebras",
      Headline => "Data type for DG algebras",
-     Version => "1.1.0",
+     Version => "1.1.1--with fix of killCycles and new code and docs for displayBlockDiff and blockDiff",
```

Comment:
FYI, this has the following effect:
![image](https://github.com/Macaulay2/M2/assets/3147718/431192da-28ea-48fa-93be-00cf311175f5)
I suspect this wasn't intended?